### PR TITLE
Automate homebrew formula upgrade process

### LINKED ISSUE #683: Auto update homebrew tflint
Hi! I noticed that @chenrui333 has been manually making the PR to homebrew whenever tflint is upgraded which can be done using a command like this.

```bash
brew bump-formula-pr \
  --strict \
  --tag=GIT_TAG_GOES_HERE \
  --revision=GIT_COMMIT_GOES_HERE \
  tflint
```

Is there a way to automate this in github actions to run this immediately when a new release is published?

Reference: https://github.com/Homebrew/homebrew-core/pull/51990

### PR DESCRIPTION
**Pull request by chenrui333**
Fixes #683

relates to https://github.com/Homebrew/homebrew-core/pull/52008

### CODE REVIEW

File: .github/workflows/homebrew.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  homebrew:
+    name: Bump Homebrew formula
+    runs-on: ubuntu-latest
+    steps:
+      - uses: mislav/bump-homebrew-formula-action@v1
+        with:
+          # A PR will be sent to github.com/Homebrew/homebrew-core to update this formula:
+          formula-name: tflint
+        env:
+          COMMITTER_TOKEN: ${{ secrets.COMMITTER_TOKEN }}
```
Comment (nitrocode) [inline .github/workflows/homebrew.yml]: Does this secret already exist?

Comment (chenrui333) [reply .github/workflows/homebrew.yml]: Not quite yet, I am working with @wata727 to set this up.

Comment (wata727) [reply .github/workflows/homebrew.yml]: Yeah, just I completed this setup.

Comment (chenrui333) [reply .github/workflows/homebrew.yml]: Thanks!!

**Comment by chenrui333** [comment]
it works, 🎉 
`https://github.com/Homebrew/homebrew-core/pull/52564`

**Pull request merged**